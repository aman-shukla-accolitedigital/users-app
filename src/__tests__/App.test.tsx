
describe('acc-api', () => {
    test('placeholder test that always passes', () => {
      expect(true).toBe(true);
    });
  });
