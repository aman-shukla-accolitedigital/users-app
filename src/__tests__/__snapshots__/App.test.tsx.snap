// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<p>
  Loading...
</p>
`;

exports[`changes the class when hovered 2`] = `
<p>
  Loading...
</p>
`;
